### PR TITLE
Add Alembic upgrade service

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: geoportal/c2cgeoportal_geoportal/scaffolds/nondockerupdate/+dot+upgrade.yaml_tmpl [hunk 1]
```diff
@@ -99,7 +99,7 @@ files_to_move:
     to: vars.yaml
     version: 2.3
   - from: alembic.ini.mako
-    to: alembic.ini
+    to: geoportal/alembic.ini
```
Comment (pvalsecc) [inline geoportal/c2cgeoportal_geoportal/scaffolds/nondockerupdate/+dot+upgrade.yaml_tmpl]: Without `.mako`?

File: geoportal/c2cgeoportal_geoportal/scaffolds/update/+dot+upgrade.yaml_tmpl [hunk 2]
```diff
@@ -142,7 +142,7 @@ files_to_move:
     to: Makefile
     version: 2.3
   - from: alembic.ini.mako
-    to: alembic.ini
+    to: geoportal/alembic.ini
```
Comment (pvalsecc) [inline geoportal/c2cgeoportal_geoportal/scaffolds/update/+dot+upgrade.yaml_tmpl]: Without `.mako`?

Comment (sbrunner) [reply geoportal/c2cgeoportal_geoportal/scaffolds/update/+dot+upgrade.yaml_tmpl]: :-/

Comment (sbrunner) [reply geoportal/c2cgeoportal_geoportal/scaffolds/update/+dot+upgrade.yaml_tmpl]: yes without mako :-)

**Pull request merged**